### ISSUE TITLE
add synthetic data source options

### ISSUE THREAD

**Issue by ebrahimebrahim** [KitwareMedical/lungair-fhir-server#4]
from @andinet :

Synthea is a synthetic EHR data generator

https://github.com/synthetichealth/synthea

It can export data in HL7 FHIR format.

Here is the original paper that talks about the simulator
https://academic.oup.com/jamia/article/25/3/230/4098271

---

more comments by me:

Synthea does not create a fhir server for you, but rather can write json to your disk that looks exactly like what you'd get if you looked up patient data that was served via the FHIR protocol ([see here](https://github.com/synthetichealth/synthea/wiki/HL7-FHIR)). It's as though you had a FHIR server with the data in it, queried the server to get some json representations of the FHIR resources, and then saved that text json to disk. So the tool we are creating still has value added beyond what Synthea does, because our tool sets up a containerized server for you. However, our tool is less ambitious than Synthea in that we are (currently) only supporting Patients and Observations (compare to the giant list of FHIR resources seen [here again](https://github.com/synthetichealth/synthea/wiki/HL7-FHIR)).

So in conclusion I do believe it will be valuable, perhaps later, to incorporate synthea as one form of data generation that our tool can use.

**Comment by ebrahimebrahim** [comment]
Another approach to look into:

EHR-Safe: Generating High-Fidelity and Privacy-Preserving Synthetic Electronic Health Records

https://assets.researchsquare.com/files/rs-2347130/v1/3ff773a93ecf69f5d8c99313.pdf?c=1670470423

(But we are not sure if the implementation was released)
